src/Microsoft.Azure.SignalR.Management/ServiceManager.cs

```diff
@@ -54,9 +54,16 @@ public Task<IServiceHubContext> CreateHubContextAsync(string hubName)
             }
         }
 
-        public string GenerateClientAccessToken(string hubName, IList<Claim> claims, TimeSpan? lifeTime = null)
+        public string GenerateClientAccessToken(string hubName, string userId, IList<Claim> claims, TimeSpan? lifeTime = null)
```

Comment:
User Id can be optional?

Reply:
If the client connection only listens to `broadcast` and `group broadcast`, then Yes. That may be one scenario.

I think both `userId` and `claims` can be optional.